admin/services/pupil.service.js

```diff
@@ -99,4 +99,20 @@ pupilService.sortByGroup = (pupilsList, sortDirection) => {
   })
 }
 
+/**
```

Comment:
Would you prefer this to be a separete service with a separete method since it's only utilised so far in access arrangements?